TSPL.docc/ReferenceManual/Attributes.md

```diff
@@ -1866,7 +1866,87 @@ into code that calls the static methods of the result builder type:
     >> assert(builderOptional == manualOptional)
     ```
   -->
-- A code block or `do` statement
+- If the result builder implements
+  the `buildPartialBlock(first:)`
+  and `buildPartialBlock(accumulated:next:)` methods,
+  a code block or `do` statement becomes a call to those methods.
+  The first statement inside of the block
+  is transformed to become an argument
+  to the `buildPartialBlock(first:)` method,
+  and the remaining statements become nested calls
+  to the `buildPartialBlock(accumulated:next:)` method.
+  For example, the following declarations are equivalent:
+
+  ```swift
+  struct DrawBoth<First: Drawable, Second: Drawable>: Drawable {
+      var first: First
+      var second: Second
+      func draw() -> String { return first.draw() + second.draw() }
```

Comment:
I'm not sure why previous comment about `return` has been resolved but there is no need for that in all three cases because there is only one expression in the body.

Reply:
> I'm not sure why previous comment about `return` has been resolved but there is no need for that in all three cases because there is only one expression in the body.

[My previous reply](https://github.com/apple/swift-book/pull/151#issuecomment-1630291632) explains why I left this the way it is:

> > @krilnon I originally wrote the code exactly according to your suggestions, but I changed it in order to stay consistent with the style of the other code examples in the same section (as I figured changing the style of the existing examples would be arguably out of scope for this PR).

Reply:
I see, since it wasn’t in the thread I assumed it was unrelated…

Reply:
Sorry for the confusion! IIRC my thought process went "it'd be silly to repeat myself in all three suggestion comments" and somehow it didn't occur to me until later that I could have just replied to any one of them 😅

Reply:
No worries!